app/views/games/show.html.erb

```diff
@@ -1,3 +1,27 @@
 <h1>Join this Game!</h1>
 
+<h6><%= link_to 'Go back to open games', root_path %></h6>
+
 <h3><%= @game.name %></h3>
+
+<% if user_signed_in? %>
+  <% if @game.white_player_id == nil %>
```

Comment:
other options

`if @game.white_player.nil?`

Or reading negatives can be tricky. So could reverse the logic and use

`if @game.white_player`